<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dManualKomMSS</class>
 <widget class="QDialog" name="dManualKomMSS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ручное управление коммутатор МСС</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>МСС А</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QRadioButton" name="rbMSSA_osn">
       <property name="text">
        <string>Основной</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbMSSA_res">
       <property name="text">
        <string>Резервный</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>МСС Б</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QRadioButton" name="rbMSSB_osn">
       <property name="text">
        <string>Основной</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbMSSB_res">
       <property name="text">
        <string>Резервный</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>МСС В</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QRadioButton" name="rbMSSV_osn">
       <property name="text">
        <string>Основной</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbMSSV_res">
       <property name="text">
        <string>Резервный</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>МСС Г</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QRadioButton" name="rbMSSG_osn">
       <property name="text">
        <string>Основной</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbMSSG_res">
       <property name="text">
        <string>Резервный</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>МСС Д</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QRadioButton" name="rbMSSD_osn">
       <property name="text">
        <string>Основной</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbMSSD_res">
       <property name="text">
        <string>Резервный</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>641</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="rightMargin">
     <number>10</number>
    </property>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbClose">
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>dManualKomMSS</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
